ambry-account/src/test/java/com/github/ambry/account/AccountTestUtils.java

```diff
@@ -56,12 +56,13 @@ static void assertAccountsInAccountService(Collection<Account> accounts, int exp
    * @param accountService The {@link AccountService} to assert {@link Account} existence.
    */
   static void assertAccountInAccountService(Account account, AccountService accountService) {
+    account = adjustSnapshotVersion(account, 1);
```

Comment:
is the adjusting of the version required after the removal of snapshot version from `equals()`?

Reply:
Ah, probably not required anymore. I'll go through and clean up the usages of this.